packages/vkui/src/components/Skeleton/Skeleton.tsx

```diff
@@ -0,0 +1,105 @@
+import * as React from 'react';
+import { classNames } from '@vkontakte/vkjs';
+import { CSSCustomProperties, HTMLAttributesWithRootRef } from '../../types';
+import { RootComponent } from '../RootComponent/RootComponent';
+import styles from './Skeleton.module.css';
+
+export interface SkeletonProps extends HTMLAttributesWithRootRef<HTMLDivElement | HTMLSpanElement> {
+  width?: React.CSSProperties['width'];
+  height?: React.CSSProperties['height'];
+  inlineSize?: React.CSSProperties['inlineSize'];
+  blockSize?: React.CSSProperties['blockSize'];
+  maxWidth?: React.CSSProperties['maxWidth'];
+  maxInlineSize?: React.CSSProperties['maxInlineSize'];
+  borderRadius?: React.CSSProperties['borderRadius'];
+
+  /**
+   * Сделает скелетон круглым. Имеет более высокий приоритет над `borderRadius`
+   */
+  circle?: boolean;
+
+  /**
+   * Начальный цвет анимации
+   */
+  colorFrom?: string;
+
+  /**
+   * Финальный цвет анимации
+   */
+  colorTo?: string;
+
+  /**
+   * Выключает анимацию, в результате чего показывается только один цвет
+   */
+  noAnimation?: boolean;
+
+  /**
+   * Длительность анимации в секундах
+   */
+  duration?: number;
+}
+
+/**
+ * Старайтесь минимизировать количество заглушек на экране. Не каждый элемент
+ * на экране должен заменяться заглушкой.
+ *
+ * Текстовые блоки лучше сокращать до трёх строк. Ширина последней строки
+ * скелета вычисляется, как 75% от ширины текстового блока. Высота скелетона
+ * автоматически подстраивается под размер шрифта, поэтому идеально
+ * вписывается в слоты компонентов, которые обычно ожидают текст.
```

Comment:
```suggestion
 * > Старайтесь минимизировать количество заглушек на экране. Не каждый элемент
 * > на экране должен заменяться заглушкой.
 * >
 * > Текстовые блоки лучше сокращать до трёх строк. Ширина последней строки
 * > скелета вычисляется, как 75% от ширины текстового блока. Высота скелетона
 * > автоматически подстраивается под размер шрифта, поэтому идеально
 * > вписывается в слоты компонентов, которые обычно ожидают текст.
```

т.к. звучит как предупреждение

<img width="320" alt="image" src="https://github.com/VKCOM/VKUI/assets/5850354/73b2a1e2-0742-47a2-a3c3-93b9cce81660">